src/triggers.js

```diff
@@ -289,16 +302,18 @@ export function maybeRunAfterFindTrigger(triggerType, auth, className, objects,
       object.className = className;
       return Parse.Object.fromJSON(object);
     });
-    const triggerPromise = trigger(request, response);
-    if (triggerPromise && typeof triggerPromise.then === "function") {
-      return triggerPromise.then(promiseResults => {
-        if(promiseResults) {
-          resolve(promiseResults);
-        }else{
-          return reject(new Parse.Error(Parse.Error.SCRIPT_FAILED, "AfterFind expect results to be returned in the promise"));
-        }
-      });
-    }
+    return Promise.resolve().then(() => {
+      const response = trigger(request);
+      if (response && typeof response.then === 'function') {
+        return response.then((results) => {
+          if (!results) {
+            throw new Parse.Error(Parse.Error.SCRIPT_FAILED, "AfterFind expect results to be returned in the promise");
```

Comment:
`AfterFind expect` should be `AfterFind expects`

Reply:
ahah :) it was there but I'll update